### PR TITLE
Adds the militaristic captain colouriser + colourisers can have multiple uses

### PR DESCRIPTION
**Pull request by ToasterBan**
## About The Pull Request
as title says, sprites done by thatguythere03

## Why It's Good For The Game
someone wanted a more militaristic captain outfit and someone else made the sprites but doesn't know how git works so I did the git work

## Testing Photographs and Procedure

<details>
<summary>Screenshots&Videos</summary>

Military Cap drip

</details>

## Changelog
:cl: thatguythere03 for the sprites, Toasterban for the codework
add: Military-styled captain outfit acquired via a colouriser in the beecoin shop
add: colourisers now can have multiple uses
/:cl:

### CODE REVIEW

**Comment by zeskorion** [comment]
>milrp nerds
ugh

**Comment by AnCopper** [comment]
We have an issue of the Captain having like 4 sets of high grades Armour in their locker, honestly it should be replaced with a vendor.

**Comment by PestoVerde322** [comment]
As much as the spritework looks nice and all, I gotta agree with AnCopper, we've far far too many sets of high grade armor in the cap's locker, and really, we might have actually one too many outfits at this point.

**Comment by bluezorua** [comment]
alright makes sense, thoughts about making the outfit a colouriser instead?

**Comment by PestoVerde322** [comment]
> alright makes sense, thoughts about making the outfit a colouriser instead?

That might actually work, maybe as a variant to the parade outfit I think.

**Comment by bluezorua** [comment]
made it a colouriser and refracted colourisers slightly to allow multiple uses

File: code/game/objects/items/colorizers/clothing.dm [hunk 1]
```diff
@@ -13,3 +13,10 @@
     name = "Head of Security's Cloak Colorizer (Royal)"
     allowed_targets = list(/obj/item/clothing/neck/cloak/hos)
     apply_icon_state = "royalhoscloak"
+
+/obj/item/colorizer/militarycapuniform
+	name = "Captain's Clothes Colourizer (Military)"
```
Comment (EvilDragonfiend) [inline code/game/objects/items/colorizers/clothing.dm]: What's this abomination of British and American grammar?

File: code/modules/client/loadout/loadout_colorizers.dm [hunk 2]
```diff
@@ -27,3 +27,9 @@
     path = /obj/item/colorizer/iandeathsquad
     allowed_roles = list(JOB_NAME_HEADOFPERSONNEL)
     cost = 20000
+
+/datum/gear/colorizer/militarycap
+    display_name = "Captain's Clothes Colourizer (Military)"
```
Comment (EvilDragonfiend) [inline code/modules/client/loadout/loadout_colorizers.dm]: this too

**Comment by bluezorua** [comment]
wait I fucked something up

**Comment by bluezorua** [comment]
wait I fucked something up

File: code/game/objects/items/colorizers/colorizer.dm [hunk 3]
```diff
@@ -52,5 +52,6 @@
 
     to_chat(user, "<span class='notice'>Color applied!</span>")
     playsound(src, 'sound/effects/spray.ogg', 5, TRUE, 5)
-    qdel(src)
-
+    uses_left --
+    if(uses_left == 0)
```
Comment (itsmeow) [inline code/game/objects/items/colorizers/colorizer.dm]: !uses_left

File: code/game/objects/items/colorizers/clothing.dm [hunk 4]
```diff
@@ -13,3 +13,10 @@
     name = "Head of Security's Cloak Colorizer (Royal)"
     allowed_targets = list(/obj/item/clothing/neck/cloak/hos)
     apply_icon_state = "royalhoscloak"
+
+/obj/item/colorizer/militarycapuniform
```
Comment (itsmeow) [inline code/game/objects/items/colorizers/clothing.dm]: change the type path to `captain_military_uniform`

File: code/modules/clothing/head/jobs.dm [hunk 5]
```diff
@@ -43,7 +43,6 @@
 
 	dog_fashion = null
 
-
```
Comment (itsmeow) [inline code/modules/clothing/head/jobs.dm:46]: random line removal

File: code/modules/clothing/under/jobs/command.dm [hunk 6]
```diff
@@ -28,3 +28,5 @@
 	icon_state = "captain_parade"
 	item_state = "by_suit"
 	can_adjust = FALSE
+
```
Comment (itsmeow) [inline code/modules/clothing/under/jobs/command.dm]: random line addition

**Review by itsmeow — CHANGES_REQUESTED** [review]

**Review by PowerfulBacon — APPROVED** [review]

**Pull request merged**